Wikipedia/Code/WMFMainPageContentSource.m

```diff
@@ -103,6 +104,10 @@ - (void)fetchAndSaveMainPageForSiteURL:(NSURL *)siteURL intoManagedObjectContext
                             WMFContentGroup *section = [moc fetchOrCreateGroupForURL:groupURL ofKind:WMFContentGroupKindMainPage forDate:[NSDate date] withSiteURL:siteURL associatedContent:nil customizationBlock:NULL];
                             [moc fetchOrCreateArticleWithURL:data.mainPageURL updatedWithSearchResult:[results firstObject]];
                             section.content = @[data.mainPageURL];
+                            if (![[NSUserDefaults wmf_userDefaults] wmf_didMigrateMainPageDailySortPriority]) {
```

Comment:
Could you remove the new user defaults key & just `updateDailySortPriority` on every refresh? This way, if we ever change the sort priority in the future it'll be applied. The object is being updated at this point anyway so it's a low cost op.